packages/config/src/chainConfig/configs/mainnet.ts

```diff
@@ -121,4 +121,14 @@ export const chainConfig: ChainConfig = {
   MAX_BLOBS_PER_BLOCK_ELECTRA: 9,
   // MAX_REQUEST_BLOCKS_DENEB * MAX_BLOBS_PER_BLOCK_ELECTRA
   MAX_REQUEST_BLOB_SIDECARS_ELECTRA: 1152,
+
+  // Fulu
+  // `2**12` (= 4096 epochs, ~18 days)
+  MIN_EPOCHS_FOR_DATA_COLUMN_SIDECARS_REQUESTS: 4096,
+  SAMPLES_PER_SLOT: 8,
+  CUSTODY_REQUIREMENT: 4,
+  NODE_CUSTODY_REQUIREMENT: 1,
+  VALIDATOR_CUSTODY_REQUIREMENT: 8,
+  BALANCE_PER_ADDITIONAL_CUSTODY_GROUP: 32000000000,
+  MAX_BLOBS_PER_BLOCK_FULU: 12,
```

Comment:
`MAX_BLOBS_PER_BLOCK_FULU` will be removed by https://github.com/ethereum/consensus-specs/pull/4277 can still add it for now if it's needed to pass config tests

Reply:
https://github.com/ChainSafe/lodestar/issues/7786